README.md

```diff
@@ -1,6 +1,10 @@
 # GoIV, Android Project for Pokémon GO.
 
 [![Travis Build Status](https://img.shields.io/travis/farkam135/GoIV/master.svg?maxAge=2592000 "Travis Build Status")](https://travis-ci.org/farkam135/GoIV)
+[![Github All Releases](https://img.shields.io/github/downloads/farkam135/GoIV/total.svg?maxAge=2592000)](https://github.com/farkam135/GoIV/releases)
+[![GitHub release](https://img.shields.io/github/release/farkam135/GoIV.svg?maxAge=2592000)](https://github.com/farkam135/GoIV/releases)
```

Comment:
So we have these two release badges below (from @DJCrapsody), and they're linked to sth. else. I guess we want some consistency?